cashu/mint/ledger.py

```diff
@@ -927,7 +927,8 @@ async def melt(
             logger.trace("verified proofs")
 
             total_provided = sum_proofs(proofs)
-            invoice_obj = bolt11.decode(invoice)
+            invoice_obj = decode(invoice)

```

Comment:
Minor: would prefer `import bolt11` then `bolt11.decode(invoice)`, here it's a bit of a "huh which decode" when reading (e.g. I'd think of the stdlib `bytes.decode` first)

Reply:
is `from bolt11.decode import decode as bolt11_decode` ok?

Reply:
sure